content/en/docs/tasks/access-application-cluster/ingress-minikube.md

```diff
@@ -62,6 +64,22 @@ This page shows you how to set up a simple Ingress which routes requests to Serv
     storage-provisioner                         1/1       Running   0          2m
     ```
 
+    For minikube  version 1.19.0 or later run
+
+    ```shell
+    kubectl get pods -n ingress-nginx
+    ```
+    {{< note >}}This can take up to a minute.{{< /note >}}
+
+     Output:
+
+    ```shell
+    NAME                                        READY   STATUS      RESTARTS    AGE
+    ingress-nginx-admission-create-g9g49        0/1     Completed   0          11m
+    ingress-nginx-admission-patch-rqp78         0/1     Completed   1          11m
+    ingress-nginx-controller-59b45fb494-26npt   1/1     Running     0          11m
+    ```
+
```

Comment:
I would use [tabs](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#tabs) for this, with minikube v1.19 or later as the first (default) tab.

@Kartik494 would you be willing to update that?

Reply:
@sftim  sure will be updating this to used tabs